modules/audio/inference/direction_detection.h

```diff
@@ -19,21 +19,59 @@
 
 #include <algorithm>
 #include <cmath>
+#include <memory>
+#include <string>
+#include <utility>
 #include <vector>
 
+#include "Eigen/Eigen"
+// Eigen 3.3.7: #define ALIVE (0)
+// fastrtps: enum ChangeKind_t { ALIVE, ... };
+#if defined(ALIVE)
+#undef ALIVE
+#endif
+
 #include "ATen/ATen.h"
 #include "cyber/cyber.h"
+#include "modules/common/proto/geometry.pb.h"
 #include "torch/torch.h"
```

Comment:
Header orders: `torch/torch.h` should be right after `ATen/ATen.h`. And add a blank line  after it.

Reply:
Thank you I have fixed it.